docs/int/openapi.yaml

```diff
@@ -796,6 +796,196 @@ paths:
               schema:
                 $ref: '#/components/schemas/Collection'
 
+  /api/metadata/job-codes:
```

Comment:
Existing endpoint of /api/search/jobcodes so please 
- [ ] change this endpoint to /api/metadata/jobcodes for API consistency

right now. We will switch them both to 'job-codes' in the 3.0 version of the API